giforgif/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: It's pronounced GIF!
+---
+
+# It is gif or gif?
+
+<img src="https://images.duckduckgo.com/iu/?u=http%3A%2F%2Fimg.pandawhale.com%2F90837-what-is-GIF-JIF-jeopardy-YES-I-sZL0.gif&f=1" />
```

Comment:
Snag the image and host in the giforgif directory, pop. I'm paying the S3 bill so 💸